.NET/Microsoft.Recognizers.Text.DataTypes.TimexExpression/TimexParsing.cs

```diff
@@ -52,15 +52,33 @@ private static void ExtractDateTime(string s, TimexProperty timexProperty)
 
             if (indexOfT == -1)
             {
+                var indexOfP = s.IndexOf('P');
```

Comment:
Why didn't this happen before? Can you add a code comment explaining cases that fall here?
Also, change the hardcoded 'P' to reuse a constant.

Reply:
assigned  indexOfP var with indexOfT. And pushed the change.